src/lib/Tactic.ml

```diff
@@ -62,7 +62,7 @@ struct
   let prf phi = {tp = D.TpPrf phi; con = D.Prf}
   let con {tp; con} = con
   let syn {tp; con} =
-    let+ tm = EM.lift_qu @@ Qu.quote_con tp con in
+    let+ tm = EM.quote_con' tp con in
```

Comment:
This is the Bug 1. @jonsterling 

Quick Fix 1: `EM.drop_joins_and_lift_qu @@ Qu.quote_con tp con`